examples/great_expecations/ge_expect_column_values_vs_profile_min.ipynb

```diff
@@ -0,0 +1,283 @@
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "context = ge.get_context()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "data_path = \"../../dataprofiler/tests/data/csv/ds_salaries.csv\"\n",
+    "df = pd.read_csv(data_path)\n",
+    "df.head()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "In this example we are going to split up the dataset into three separate years so we can simulate a dataset which will have a yearly aggregation of data."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "df.sort_values(by=\"work_year\", axis=0, inplace=True)\n",
+    "years = df[\"work_year\"].unique().tolist()\n",
+    "years"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now that we have the years, we will capture all records from each year in their own dataframes, so we can process them separately."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "individual_dataframes = []\n",
+    "for year in years:\n",
+    "    current_year_df = df.loc[df[\"work_year\"]==year]\n",
+    "    current_year_df = current_year_df.drop(\"work_year\", axis=1)\n",
+    "    individual_dataframes.append(current_year_df)\n",
+    "individual_dataframes[0].head()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Now we will create a report on the first `individual_dataframe` which corresponds to the year 2020, then we will output the `min` metric from the `salary_in_usd` column as found in the report."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "profiler_options = dp.ProfilerOptions()\n",
+    "profiler_options.set({\"data_labeler.is_enabled\": False})\n",
+    "profile = dp.Profiler(individual_dataframes[0], len(individual_dataframes[0]), options=profiler_options)\n",
+    "report  = profile.report(report_options={\"output_format\": \"compact\"})"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
```

Comment:
Add a cell here for "Let's take a look at the output `min` metric from the `salary_in_usd` column as found in the report"